### PR TITLE
A quick fix for issue #94

### PR DESCRIPTION
**Pull request by kira-lin**
In current ray versions,  models like Sequential in pytorch, which implements Iterable interface, will be treated as a list of models. A quick work-around for this is to pass in our single model as a list, then unpack it afterwards.

### CODE REVIEW

**Comment by ConeyLiu** [comment]
Hi @kira-lin, could you add a test for this?

File: python/raydp/torch/estimator.py [hunk 1]
```diff
@@ -209,12 +209,15 @@ def setup(self, config):
                 else:
                     lr_scheduler = None
 
+                # A quick work-around for https://github.com/ray-project/ray/issues/14352
+                # As we only support single model for now, this is OK
                 registered = self.register(
-                    models=model, optimizers=optimizer, criterion=loss, schedulers=lr_scheduler)
+                    models=[model], optimizers=optimizer, criterion=loss, schedulers=lr_scheduler)
```
Comment (ConeyLiu) [inline python/raydp/torch/estimator.py:215]: should we wrap the optimizer, loss, lr_scheduler as well?

Comment (kira-lin) [reply python/raydp/torch/estimator.py:215]: No need to wrap loss. I'm not sure if any optimizer implements Iterable, but I'll wrap them anyway.

**Comment by kira-lin** [comment]
> Hi @kira-lin, could you add a test for this?

I modified the original test code a bit:
```python
import ray
import raydp
from raydp.torch import TorchEstimator
#start Spark job on Ray
ray.init(address='auto')
spark = raydp.init_spark(
    app_name = "example test",
    num_executors = 3,
    executor_cores = 3,
    executor_memory="10GB"
)
##prepare the data
customers = [
    (1,'James', 21, 6),
    (2, "Liz", 25, 8),
    (3, "John", 31, 6),
    (4, "Jennifer", 45, 7),
    (5, "Robert", 41, 5),
    (6, "Sandra", 45, 8)
]
df = spark.createDataFrame(customers, ["cID", "name", "age", "grade"])
df.show()

import torch
##create model
model = torch.nn.Sequential(torch.nn.Linear(1, 2), torch.nn.Linear(2,1))
optimizer = torch.optim.Adam(model.parameters())
loss = torch.nn.MSELoss()
#config
estimator = TorchEstimator(
    model = model,
    optimizer = optimizer,
    loss = loss,
    num_workers = 3,
    num_epochs = 5,
    feature_columns = ["age"],
    label_column = ["grade"],
    batch_size=2
)
print("before fit on spark, the type of model:", type(model))
estimator.fit_on_spark(df)
##get the trained model
pytorch_model = estimator.get_model()
```

**Comment by ConeyLiu** [comment]
You can add it to the `tests` folder.

**Comment by ConeyLiu** [comment]
thanks.
